[Site] Change color and style of menu items --signoff

NikhilSharma03 opened the pull request:
Signed-off-by: Nikhil <nikhilsharmamusic2000@gmail.com>

**Description**
Updated faq color and style as in the Figma design

## After Changes
![FAQ](https://user-images.githubusercontent.com/58226527/120435327-35822180-c39b-11eb-8bf6-13fe1fe09e05.png)

This PR fixes #124 

**Notes for Reviewers**
@JubayerJoy Can you please confirm the changes so I can work on the " The sidebar of Getnighthawk Docs " section too? Thank you.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

vinayaksh42 commented:
Nice changes @NikhilSharma03

JubayerJoy reviewed (commented):
Nice work @NikhilSharma03, the implementation is near perfect as the design 🎈 

Needs some fixes,
1. The `font-family` of this section is specified to be `open sans` in the design.
2. There are some issues with the smaller screen:
![Screenshot (216)](https://user-images.githubusercontent.com/31549564/120483114-9cbbc800-c3d3-11eb-8179-1672d8eb2fd8.png)
 #### Expected: 
 
![Screenshot (217)](https://user-images.githubusercontent.com/31549564/120483544-063bd680-c3d4-11eb-969d-c567fb775e42.png)

Good luck 👍

NikhilSharma03 commented:
Thank you for the review @JubayerJoy. I will work on that.

JubayerJoy commented:
@NikhilSharma03 You can do a similar implementation for the Docs menu, it can be done in this or separate PR at your convenience. Thank you!

NikhilSharma03 commented:
Yes for sure @JubayerJoy. I was just making sure if these changes are correct so I can implement these changes into the docs too.
I guess a different PR will be more right since the changes belong to a different section.

NikhilSharma03 commented:
## Changes Made
- Updated font-family to Open Sans for this section
- Fixed padding of the menu

## After Changes
![Screenshot from 2021-06-02 20-17-53](https://user-images.githubusercontent.com/58226527/120502715-4903ab80-c3e0-11eb-8e25-2afbb4f50d51.png)

@JubayerJoy Can you please review the changes?

JubayerJoy commented on docs/_sass/faq.scss:
```diff
@@ -27,4 +66,12 @@
+
+@media(max-width:1100px){
+    .container-faq .col-lg-8{
+        padding-left: 4rem;
+        padding-right: 4rem;
```
Let's trim down the padding a bit more for matching this section with the upper ones.

```suggestion
        padding-left: 2rem;
        padding-right: 2rem;
```

NikhilSharma03 replied:
Yeah Sure.

JubayerJoy reviewed (commented):
@NikhilSharma03 Really nice work 🚀
Just a small change....

NikhilSharma03 commented:
@JubayerJoy Can you please review? Thank you.

JubayerJoy reviewed (approved):
LGTM 🎈

NikhilSharma03 commented:
Thank you @JubayerJoy. @vinayaksh42 Can you please review the changes? Thank you.

leecalcote commented:
Some good work happening here.

NikhilSharma03 commented:
@leecalcote 😂🚀🚀
